login-workflow/docs-new/language-support.md

```diff
@@ -8,6 +8,28 @@ This package supports translations to different languages using [i18next](https:
 -   Spanish
 -   Simplified Chinese
 
-## Changing the Language
+## Add/Override translations with our workflow
 
-<!-- @TODO: Finish this... -->
\ No newline at end of file
+To add i18n in your application you can follow the instructions from [official website](https://react.i18next.com/getting-started).
+
+If you want to add/override your custom screens translations in our `Auth Workflow` you need to pass your app's `i18n instance` to `AuthContextProvider` via prop name `i18n`.
+
+```tsx
+import { i18nAppInstance } from './i18nAppInstance';
```

Comment:
Should we have a small example of what this i18nAppInstance would look like? Or is it thoroughly documented in the i18n docs so we don't need an example here?

Reply:
Seems thoroughly documented through the reference link you provide so this isn't necessary